pallets/attestation/src/lib.rs

```diff
@@ -465,6 +471,19 @@ pub mod pallet {
 				ExternalAttestations::<T>::remove(authorization_id, claim_hash);
 			}
 		}
+
+		#[cfg(any(feature = "try-runtime", test))]
+		pub fn do_try_state() -> Result<(), &'static str> {
+			Attestations::<T>::iter().try_for_each(|(claim_hash, attestation_details)| -> Result<(), &'static str> {
+				if let Some(authorization_id) = attestation_details.authorization_id {
+					ensure!(
+						ExternalAttestations::<T>::get(authorization_id, claim_hash),
+						"Unknown external attestation"
```

Comment:
What about including some more information in the error strings? Something like `format!("External attestation entry does not exist for attestation with authorization ID {:?}", authorization_id)`. This will save us a lot of time in case something does actually break in the future. Probably the same could be applied to most of the error messages.